docs/api/web-frame.md

```diff
@@ -74,6 +74,17 @@ Sets the maximum and minimum layout-based (i.e. non-visual) zoom level.
 
 Sets a provider for spell checking in input fields and text areas.
 
+If you want to use this method you must disable the builtin spellchecker when you
```

Comment:
Is this section needed in 8-x-y since the default of `spellcheck` is already `false`?

Reply:
Left it in to be clear that the two settings are incompatible.